misc/toxic.conf.example

```diff
@@ -169,6 +169,23 @@ groupchats = {
 // };
 };
 
+// Conference-specific config settings. Public keys are used as group identifiers
+// and must begin with the prefix "pk_". These take priority over global settings.
+conferences = {
+  pk_PUBLIC_KEY_1 = {
```

Comment:
What will happen if this is an invalid public key? I guess "config error: failed to set conference tab name color"? I wonder if the `PUBLIC_KEY_1` should be some valid (but non-existent) public key like `AAAAAA...` or `CAFEBABEDEADBEEFFEEDB00B`?

Reply:
It'll give the same stderr warning whether it's a fake public key, a random name, or a real public key that isn't found in your profile.

Reply:
Also I think in a different commit I'll make those errors more clear, so you can determine if the problem is an invalid pk or something else.